File: godotopenxrlynx/src/main/java/org/godotengine/openxrloaders/lynx/GodotOpenXRLynx.java
package org.godotengine.openxrloaders.lynx;

import org.godotengine.godot.Godot;
import org.godotengine.godot.plugin.GodotPlugin;

/**
 * \brief GodotOpenXRLynx is the OpenXR Lynx loader plugin for Godot.
 */
public class GodotOpenXRLynx extends GodotPlugin {

    public GodotOpenXRLynx(Godot godot) {
        super(godot);
    }

    @Override
    public String getPluginName() {
        return "GodotOpenXRLynx";
    }
}

File: godotopenxrlynx/src/main/java/org/godotengine/openxr/vendors/lynx/GodotOpenXRLynx.kt
package org.godotengine.openxr.vendors.lynx

import org.godotengine.godot.Godot
import org.godotengine.godot.plugin.GodotPlugin

/**
 * \brief GodotOpenXRLynx is the OpenXR Lynx plugin for Godot.
 */
class GodotOpenXRLynx(godot: Godot?) : GodotPlugin(godot) {
    override fun getPluginName(): String {
        return "GodotOpenXRLynx"
    }
}
